src/drive/web/modules/trash/Toolbar.jsx

```diff
@@ -38,18 +34,16 @@ export const Toolbar = ({
 
   const { pushModal, popModal } = useContext(ModalContext)
 
-  const onEmptyTrash = flag('drive.client-migration.enabled')
-    ? useCallback(() => {
-        pushModal(
-          <EmptyTrashConfirm
-            onClose={popModal}
-            onConfirm={() => {
-              client.collection('io.cozy.files').emptyTrash()
-            }}
-          />
-        )
-      })
-    : useCallback(emptyTrash)
+  const onEmptyTrash = useCallback(() => {
+    pushModal(
+      <EmptyTrashConfirm
+        onClose={popModal}
+        onConfirm={() => {
+          client.collection('io.cozy.files').emptyTrash()
```

Comment:
This should be awaited ?

Reply:
What happens if there is an error during emptying of the trash ?

Reply:
Good catch, the migration on this is incomplete.